components/brave_ads/README.md

```diff
@@ -6,29 +6,36 @@ Users earn tokens by viewing privacy-first Brave Ads. Ads presented are based on
+| Advertiser  | An advertiser is someone or a company that promotes products, services, or ideas through various mediums to reach and persuade potential customers or target audiences.  |
+| Anti-targeting  | Exclude users from receiving targeted advertisements based on specific attributes such as demographics, interests, or behaviors.  |
+| Behavioral  | Behavioral advertising targets ads based on the user's previous online behavior, such as browsing history and interactions, to deliver relevant and personalized advertisements.  |
+| Campaign  | A campaign is a coordinated series of marketing activities and messages designed to achieve specific goals within a defined timeframe.  |
+| Catalog  | Inventory of available advertisements.  |
+| Click-through rate  | The percentage of ad impressions that result in clicks.  |
+| Confirmations  | See [security and privacy model for ad confirmations](https://github.com/brave/brave-browser/wiki/Security-and-privacy-model-for-ad-confirmations).  |
+| Contextual  | Contextual advertising targets ads based on the web page's content or the user's online activity context to deliver relevant and personalized advertisements.  |
+| Conversion  | The desired view-through or click-through action a user takes in response to an ad, such as making a purchase or filling out a form.  |
+| Creative  | A creative refers to a specific version or variation of an advertisement allowing advertisers to test different approaches, optimize performance, and effectively engage their target audience.  |
+| Creative set  | A creative set refers to a collection of related creative instances or variations of an advertisement.  |
+| Day parting  | Day parting is a strategy that involves scheduling and targeting ads based on specific times of the day to maximize their effectiveness.  |
+| Delivery  | Delivering ad content to the user, also known as serving.  |
+| Frequency capping  | Limits how many times a given ad will be shown within a specified period, also known as exclusion rules.  |
+| Geographic  | Geographic advertising targets and tailors ads based on specific geographic locations to reach the desired audience.  |
+| Impression  | An impression is when an ad is displayed or shown to a user, also known as a view.  |
+| Multi-armed bandits  | Multi-armed bandits optimize the allocation of resources between different ad variations or strategies to maximize performance.  |
+| Pacing  | Rate at which an ad campaign uses up its pre-set number of impressions.  |
+| Placement  | The location at which an ad is displayed to the user.  |
+| Priority  | Priority at which an ad campaign uses up its impressions.  |
+| Purchase intent  | Likelihood or inclination of a consumer to make a purchase based on their expressed interest or behavior. It indicates the consumer's readiness to buy a product or service.  |
+| Resource  | Resources provided by [components](brave://components).  |
```

Comment:
nit: `Ads Resources`?

Reply:
I can, but I decided to remove Ads from other terms because this file is only for Ads. Your thoughts?

Reply:
Fixed, thanks